https://bugs.gentoo.org/943696#c10
https://github.com/alsa-project/alsa-lib/commit/07cee0ba05179a56764c35975d5822420d4f31f9

From 07cee0ba05179a56764c35975d5822420d4f31f9 Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Sun, 24 Nov 2024 09:33:16 +0100
Subject: [PATCH] seq: include UMP headers

Some applications seem including alsa/seqmid.h individually, and this
got broken with the update of alsa-lib because now we have
dependencies to UMP stuff.  Include the necessary UMP headers
internally.  Also, add the inclusion of rawmidi.h in ump.h for similar
reasons.

Link: https://bugzilla.suse.com/show_bug.cgi?id=1233682
Signed-off-by: Takashi Iwai <tiwai@suse.de>
--- a/include/seq.h
+++ b/include/seq.h
@@ -29,6 +29,8 @@
 #ifndef __ALSA_SEQ_H
 #define __ALSA_SEQ_H
 
+#include "ump.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
--- a/include/seq_event.h
+++ b/include/seq_event.h
@@ -28,6 +28,8 @@
 #ifndef __ALSA_SEQ_EVENT_H
 #define __ALSA_SEQ_EVENT_H
 
+#include "ump_msg.h"
+
 /**
  *  \defgroup SeqEvents Sequencer Event Definitions
  *  Sequencer Event Definitions
--- a/include/ump.h
+++ b/include/ump.h
@@ -9,6 +9,8 @@
 #ifndef __ALSA_UMP_H
 #define __ALSA_UMP_H
 
+#include "rawmidi.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif

